src/module/apps/characterImport/vehicleImport/VehicleParser.js

```diff
@@ -18,12 +18,17 @@ export default class VehicleParser {
 
         const vehicles = getArray(chummerChar.vehicles?.vehicle);
 
-        for(let vehicle of vehicles) {
+        vehicles.forEach(async vehicle => {
```

Comment:
forEach with async will create a detached thread for each call. Since I am gathering them in an array that is attached to the main thread awaiting them later is no issue

Reply:
My point wasn't that it doesn't work, or how. My point was the potential pitfall that not everyone might recognize of forEach not awaiting the result. :) JS istn't all that clear for that fact sometimes.

(Just a fyi) Btw, async/await isn't multithreaded. It's asynchronous. JavaScript has one thread per runtime, which means web workers are the only support for multi threading it has.